plugins/org.jkiss.dbeaver.ext.altibase/OSGI-INF/l10n/bundle.properties

```diff
@@ -193,6 +199,7 @@ meta.org.jkiss.dbeaver.ext.altibase.model.session.AltibaseServerSession.utransTi
 meta.org.jkiss.dbeaver.ext.altibase.model.session.AltibaseServerSession.utransTimeLimit.name              = Update Transaction Timeout (sec.)
 
 tree.administer.node.name          = Administer
+tree.altibaseProperties.node.name  = V$PROPERTY
```

Comment:
Do we really want it to be named this way? Can't we use "Property" instead?

Reply:
The reason it is named V$Property is to inform the user that it is taken from a performance view called V$Property. If it is better to use a more common name, Property would be good.

Reply:
In my opinion, it's too technical for a regular user. Therefore, in this particular case, a more common name is the way to go.

Reply:
Okay, thanks for your opinion :)